docs/doc/30-reference/30-sql/10-dml/dml-copy-into-table.md

```diff
@@ -38,19 +38,48 @@ internalStage ::= @<internal_stage_name>[/<path>]
+```
+
+| Parameter                 	| Description                                                                 	| Required 	|
+|---------------------------	|-----------------------------------------------------------------------------	|----------	|
+| `s3://<bucket>/[<path>]`    	| External files located at the AWS S3 compatible object storage.             	| Optional 	|
+| ENDPOINT_URL              	| The bucket endpoint URL starting with "https://".                                  	| Optional 	|
+| ACCESS_KEY_ID             	| Your access key ID for connecting the AWS S3 compatible object storage.     	| Optional 	|
+| SECRET_ACCESS_KEY         	| Your secret access key for connecting the AWS S3 compatible object storage. 	| Optional 	|
+| REGION                    	| AWS region name. For example, us-east-1.                                    	| Optional 	|
+| ENABLE_VIRTUAL_HOST_STYLE 	| If you use virtual hosting to address the bucket, set it to "true".                               	| Optional 	|
+
+Azure Blob storage：
+```sql
+externalLocation ::=
+  'azblob://<bucket>[/<path>]'
+  CONNECTION = (
+        ENDPOINT_URL = 'https://<endpoint-URL>'
+        ACCOUT_NAME = '<your-account-name>'
+        ACCOUNT_KEY = '<your-account-key>')
 ```
 
-| Parameters                                                                                  | Description                                                                                                             | Required  |
-|---------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------|-----------|
-| `s3://<bucket>/[<path>]`                                                                    | Files are in the specified external location (S3-like bucket)                                                           | YES       |
-| `[ { CREDENTIALS = ( {  { AWS_KEY_ID = '<string>' AWS_SECRET_KEY = '<string>' } } ) } ]' ]` | The credentials for connecting to AWS and accessing the private/protected S3 bucket where the files to load are staged. | Optional  |
-| `[ ENDPOINT_URL = '<endpoint_url>' ]`                                                       | S3-compatible endpoint URL like MinIO, default is `https://s3.amazonaws.com`                                            |  Optional |
+| Parameter                  	| Description                                              	| Required 	|
+|----------------------------	|----------------------------------------------------------	|----------	|
+| `azblob://<bucket>/[<path>]` 	| External files located at the Azure Blob storage.        	| Required 	|
+| ENDPOINT_URL               	| The container endpoint URL starting with "https://".     	| Optional 	|
+| ACCOUNT_NAME               	| Your account name for connecting the Azure Blob storage. 	| Optional 	|
```

Comment:
Please note here: If `ACCOUNT_NAME` is not provided, the databend will access the bucket as anonymous users.